profiles/bicycle.lua

```diff
@@ -320,6 +320,7 @@ function way_function (way, result)
   local foot_backward = way:get_value_by_key("foot:backward")
   local bicycle = way:get_value_by_key("bicycle")
 
+  local allowed_way_type = false
```

Comment:
from the name alone I wouldn't know if that `allowed` is allowed for cycling, pushing or something different. This is cycling, right? If so, I don't understand why we allow railway platforms in the same way we do with parking areas and so on. This is, I guess, inferred from the speeds? Still I feel the name is a bit confusing in that context. I don't know how I would name it to make it clear. Maybe add some commentary?